metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -686,7 +686,7 @@ def get_parser(
             parser = self.DebeziumParser(
                 source_platform="mssql",
                 server_name=self.get_server_name(connector_manifest),
-                database_name=connector_manifest.config.get("database.dbname"),
+                database_name=connector_manifest.config.get("database.names") or connector_manifest.config.get("database.dbname"),
```

Comment:
I believe this will work only if database.names has single database name, whereas database.names supports specifying multiple comma separated database names. Is that correct understanding ?

Reply:
Now that you mention it, I think that's right. In that specific case, you wouldn't be able to discern to which database the table belongs to with the information available in the manifest. If you picked only one of them, it could lead to erroneous lineage relationships.

Since the property `database.dbname` isn't available anymore for the SQL Server connector, could we support the single DB use case? If more than one DBs are detected, maybe a warning could be shown and the parameter `database_name` could be left empty.

Reply:
Yes, can do that. One small change suggested -  when more than one DBs are detected, raise an exception, it'll get caught in the outer try..except block where we initialize DebeziumSourceConnectorLineages per connector. This will automatically report a warning . If we leave database_name empty, it'll generate incorrect table urns, without database name.

That being said, what do you think, should be ideal behavior if multiple database names are specified in connector ? What are the resolved topics in case multiple database.names (db1,db2) are specified, and both of databases have same schema name(schema1) and same table name(table1) inside it. Are changes to both tables (db1.schema1.table1 and db2.schema1.table1) written to same kafka topic or different ones ?

Reply:
Mmmh... I'm not sure, I haven't encountered that use case yet. Reading the docs it seems they'd be written to the same topic which follows the naming scheme `<topicPrefix>.<schemaName>.<tableName>`. Maybe it expects the table to exist in boths DBs and follow the same naming schemas.

How should that be translated on the lineage? Should we split the DBs and create a new lineage for each combination with the tables? 

For the moment, could we finish this fix and setup a new PR with the proper scope?

Reply:
that makes sense to me.

Reply:
Ahh, my bad, the try..except is not present in parsing block of [`DebeziumSourceConnector._extract_lineages`](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py#L710-L732) but only present in [`ConfluentS3SinkConnector._extract_lineages`](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py#L932-L951)

Could you please add this try..except in DebeziumSourceConnector and then I can approve the PR.

Reply:
Sure thing! Just pushed the changes.